CONTRIBUTING.md

```diff
@@ -114,6 +123,49 @@ Examples of PR titles:
  - test: Add infinity test for `MoveEffect.to`
  - refactor: Optimize the structure of the game loop
 
+
+# Maintainers
+
+These instructions are for the maintainers of Flame.
+
+## Merging a pull request
+
+When merging a pull request, make sure that the title of the merge commit has the correct
+conventional commit tag and a descriptive title. This is extra important since sometimes the title
+of the PR doesn't reflect what GitHub defaults to for the merge commit title (if the title has been
+changed during the life time of the PR for example).
+
+All the default text should be removed from the commit message and if the PR is breaking, the
+instructions from the "Migration instruction" should be copied into the commit message.
+
+
+## Breaking changes
+
+When doing breaking changes a deprecation tag should be added when possible and contain a message
+that conveys to the user what which version that the deprecated method/field will be removed in and 
+what method they should use instead to perform the task. The version specified should be at least
+two versions after the current one, such that there will be at least one stable release where the
+users get to see the deprecation warning and in the version after that (or a later version) the
+deprecated entity should be removed.
+
+
+## Creating a release
+
+There are a few things to think about when doing a release:
+ 
+ - Go through the codebase and look for deprecated methods/fields and remove the ones that are
```

Comment:
Perhaps searching for `@Deprecated` is easier

Reply:
Haha I don't think anyone would go through file by file and do it manually, but i can update it if you think is unclear :smile: